megamek/src/megamek/common/Compute.java

```diff
@@ -4957,6 +4958,14 @@ public static int getSensorRangeByBracket(Game game, Entity ae, @Nullable Target
 
     }
 
+    public static int getADARangeModifier(int distance) {
+        // +0 for same ground map / Low-Altitude hex
+        // +2 for 1 LAH away
+        // +4 for 2 LAH away
+        return (distance / Board.DEFAULT_BOARD_HEIGHT * 2);
```

Comment:
I'm not sure that the comment matches the code

Reply:
Wretched off-by-ones!  Fixed.